AmmadAmjad-CodeTest/Server/endpoints/comments/routes/index.js

```diff
@@ -0,0 +1,207 @@
+const {Router} = require('express');
+const app = Router();
+const underscore = require('underscore');
+const db = require('../../../controllers/db.js');
+const middleware = require('../../../controllers/middleware.js')(db);
+
+
+
+app.post('/comment/create' , middleware.requireAuthentication , function(req , res){
+    var body = underscore.pick(req.body, 'title', "description");
+    body.authorId = req.user.id;
+    db.comment.create(body)
+    .then(function(comment){
+      res.json({
+        message : "Comment added successfully",
+        comment : comment
+      });
+    })
+    .catch(function(e){
+      res.status(403).json({
```

Comment:
Error handling, it could be the user's input has issue or server failed to create the comment, so might be better to treat them differently with 4xx and 5xx codes.